cluster/examples/kubernetes/ceph/upgrade-from-v1.1-crds.yaml

```diff
@@ -286,40 +262,6 @@ spec:
 ---
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
-metadata:
```

Comment:
I removed the CRDs from this yaml that don't have an update to the schema since 1.1